.github/issue-labeler.yml

```diff
@@ -0,0 +1,146 @@
+assessment:
+  - assessment
+  - bonus
+  - complaint
+  - submission
+  - feedback
+  - grading
+  - grade
+  - score
+  -
+  - caseSensitive: false
+
+athena:
+  - athena
+  - caseSensitive: false
+
+atlas:
+  - atlas
+  - competency
+  - competencies
+  - relation
+  - source
+  - knowledge area
+  - competency catalog
+  - learning path
+  - learner profile
+  - competency catalog
+  - science event
+  - caseSensitive: false
+
+buildagent:
+  - buildagent
+  - build agent
+  - buildjob
+  - build job
+  - build result
+  - caseSensitive: false
+
+communication:
+  - communication
+  - conversation
+  - notification
+  - agreement
+  - faq
+  - post
+  - reaction
+  - caseSensitive: false
+
+core:
+  - user-management
+  - course
+  - authority
+  - data export
+  - migration
+  - user
+  - group
+  - caseSensitive: false
+
+exam:
+  - exam
+  - exercisegroup
+  - student exam
+  - suspicious
+  - caseSensitive: false
+
+exercise:
+  - exercise
+  - participation
+  - participant
+  - difficulty
+  - lifecycle
+  - submission
+  - team
+  - assignment
+  - caseSensitive: false
+
+fileupload:
+  - fileupload
+  - upload
+  - caseSensitive: false
+
+iris:
+  - iris
+  - llm
+  - caseSensitive: false
+
+lecture:
+  - lecture
+  - attachment
+  - online
+  - slide
+  - video
+  - caseSensitive: false
+
+lti:
+  - lti
+  - online course
+  - caseSensitive: false
+
+modeling:
+  - modeling
+  - diagram
+  - uml
+  - caseSensitive: false
+
+plagiarism:
+  - plagiarism
+  - caseSensitive: false
+
+programming:
+  - programming
+  - build
+  - build job
```

Comment:
This is an overlap with `buildagent` and you don't have the version without the space here, is that intended?